ssh/server/server.go

```diff
@@ -25,16 +25,6 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-var (
```

Comment:
I don't like the idea of dropping those are it makes harder to find which errors we return.